tests/test_bgt.py

```diff
@@ -1,5 +1,72 @@
-from bgt import __version__
+
+import requests
+import pytest
+import boto3
+import os
+from random import randint
+from moto import mock_dynamodb2
+from json import load, dump
+from starlette.testclient import TestClient
+from app import __version__
+from app.main import app
+from app.schema import Game
+from app.utils import take, UniversalEncoder
+
+client = TestClient(app)
+
+
+def rel_path(fn):
+    return os.path.join(os.path.dirname(__file__), fn)
+
+
+def local_url(path):
+    return "http://localhost:8000"+path
 
 
 def test_version():
     assert __version__ == '0.1.0'
+
+
+def test_root():
+    response = client.get("/")
+    assert response.status_code == 200
+
+
+@mock_dynamodb2
+def test_get_game():
+    # load table scheme
+    scheme_fp = rel_path("gameTableScheme.json")
```

Comment:
You might consider turning some of these "test inputs" into fixtures that can be reused in any number of tests. Here's an example (also note the use of pathlib):

```
from pathlib import Path

@pytest.fixture()
def scheme():
  with open(Path(__file__).parent / "gameTableScheme.json", "r") as fp:
    value = load(fp)
  return value

def test_one(scheme):
  …

def test_two(scheme):
  …
```